app/scripts/views/mixins/signin-mixin.js

```diff
@@ -24,13 +31,15 @@ define(function (require, exports, module) {
      * @return {object} promise
      */
     signIn: function (account, password) {
+      var self = this;
+      self.logEvent(`flow.${this.signInSubmitContext}.submit`);
```

Comment:
> When will client side created flow events be used instead of server side created events?

Whenever the thing represented by the event occurs client-side.

The point of these events is to help us identify where users might be dropping off during the sign-in flow. Having extra events on the client, in between the existing `flow.begin` (client) and `account.login` (server) flow events provides a finer granularity for analysing those drop-offs. FOr instance, if some imaginary future UI change caused people to stop submitting the form for some reason, these events would help us identify that.